### ISSUE TITLE
pyside is broken

### ISSUE THREAD

**Issue by tacaswell** [matplotlib/matplotlib#5196]
```
6a74cda44d8f467b4d5d2ac25e6ce7ac1efe3852 is the first bad commit
commit 6a74cda44d8f467b4d5d2ac25e6ce7ac1efe3852
Author: Peter Würtz <pwuertz@gmail.com>
Date:   Wed Aug 19 16:38:43 2015 +0200
    Qt: Redraw agg buffer in main thread. Implement FigureCanvasQTAggBase init.
:040000 040000 3d744fa5805d7eba6843dbf23b5443c1870996d0 aa5a2bfe9b814d8d79d4381be6f2d63c35a70e56 M	lib

```

6a74cda44d8f467b4d5d2ac25e6ce7ac1efe3852 went is as part of https://github.com/matplotlib/matplotlib/pull/4972

**Comment by tacaswell** [comment]
attn @pwuertz 

I don't have time tonight to dig into this beyond bisecting it.

Suspect that is is related to the pyqt4 segfaults in #5195

**Comment by tacaswell** [comment]
I did end up coming back to this and have a possible fix tacked onto #5195

**Comment by pwuertz** [comment]
I am sorry, still on vacation this week without proper means for giving you a second opinion :/
